src/components/Header/Header.stories.js

```diff
@@ -0,0 +1,19 @@
+import {MemoryRouter} from 'react-router-dom';
+
+import Header from './Header.js';
+
+export default {
+  title: 'Components/Header',
+  component: Header,
+  decorators: [
+    Story => (
+      <MemoryRouter>
+        <Story />
+      </MemoryRouter>
```

Comment:
The same for here too. 🤔 Or did you have problems without `MemoryRouter` ? There is also the possibility to set the MemoryRouter for Storybook globally via the `preview.js`. Then it is not needed to import it for every file separately. [See here for an example.](https://github.com/mbosselmann/color-saver-template/blob/main/.storybook/preview.js)